tests/unit/remoteworker/test_remote_worker.py

```diff
@@ -0,0 +1,126 @@
+import mock
+
+from unittest import TestCase
+
+from scripts.workers.remote_submission_worker import (
+    make_request,
+    get_message_from_sqs_queue,
+    delete_message_from_sqs_queue,
+    get_submission_by_pk,
+    get_challenge_phases_by_challenge_pk,
+    get_challenge_by_queue_name,
+    get_challenge_phase_by_pk,
+    update_submission_data,
+    update_submission_status,
+    return_url_per_environment,
+)
+
+
+@mock.patch("scripts.workers.remote_submission_worker.AUTH_TOKEN", "test_token")
+@mock.patch("scripts.workers.remote_submission_worker.requests")
+class MakeRequestTestClass(TestCase):
+
+    def test_make_request_get(self, mock_make_request):
+        url = "/test/url"
+        headers = {"Authorization": "Token test_token"}
+        make_request(url, "GET")
+        mock_make_request.get.assert_called_with(url=url, headers=headers)
+
+    def test_make_request_put(self, mock_make_request):
+        url = "/test/url"
+        headers = {"Authorization": "Token test_token"}
+        data = {"test": "data"}
+        make_request(url, "PUT", data=data)
+        mock_make_request.put.assert_called_with(url=url, headers=headers, data=data)
+
+    def test_make_request_patch(self, mock_make_request):
+        url = "/test/url"
+        headers = {"Authorization": "Token test_token"}
+        data = {"test": "data"}
+        make_request(url, "PATCH", data=data)
+        mock_make_request.patch.assert_called_with(url=url, headers=headers, data=data)
+
+
+@mock.patch("scripts.workers.remote_submission_worker.return_url_per_environment")
+@mock.patch("scripts.workers.remote_submission_worker.make_request")
+class APICallsTestClass(TestCase):
+
+    @mock.patch("scripts.workers.remote_submission_worker.QUEUE_NAME", "evalai_submission_queue")
+    def test_get_message_from_sqs_queue(self, mock_make_request, mock_url):
+        url = "/api/jobs/challenge/queues/evalai_submission_queue/"
+        get_message_from_sqs_queue()
+        mock_url.assert_called_with(url)
```

Comment:
make `url` as a class variable otherwise there is no use of classes plus there is lot of redundant code

Reply:
Done!